pkg/lib/interaction/nodes/use_identity_oauth_user_info.go

```diff
@@ -85,14 +85,15 @@ func (e *EdgeUseIdentityOAuthUserInfo) Instantiate(ctx *interaction.Context, gra
 		return nil, err
 	}
 
-	providerID := userInfo.ProviderConfig.ProviderID()
+	providerConfig := oauthProvider.Config()
+	providerID := providerConfig.ProviderID()
 	spec := &identity.Spec{
 		Type: authn.IdentityTypeOAuth,
 		Claims: map[string]interface{}{
 			identity.IdentityClaimOAuthProviderKeys: providerID.Claims(),
-			identity.IdentityClaimOAuthSubjectID:    userInfo.ProviderUserInfo.ID,
+			identity.IdentityClaimOAuthSubjectID:    userInfo.StandardAttributes.Sub(),
 			identity.IdentityClaimOAuthProfile:      userInfo.ProviderRawProfile,
-			identity.IdentityClaimOAuthClaims:       userInfo.ProviderUserInfo.ClaimsValue(),
+			identity.IdentityClaimOAuthClaims:       userInfo.StandardAttributes.ToClaims(),
```

Comment:
In the OAuth Claims, the original data from `ProviderUserInfo.ClaimsValue()` doesn't have `sub`. But after update, `ToClaims()` returns all values which include `sub`. For data consistent, maybe we should 

1. Remove `sub` in the `ToClaims()`
2. Handle it in the migration task later, add back `sub` for the old data

Reply:
Actually the `sub` claim is not very important, since the column `provider_user_id` is used instead. I prefer we handle data migration in a later task.